best-practices/add_network_policy/kuttl-test/02-install-resource.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
You can rename the directory to `e2e` from `kuttl-test`. Kuttl is only a way to do e2e tests.